Interfaces/API/Dirac.py

```diff
@@ -735,13 +736,14 @@ def runLocal(self, job):
       self.log.error("Could not extract job parameters from job")
       return res
     parameters = res['Value']
+    self.log.debug("Extracted job parameters from JDL", parameters)
 
     arguments = parameters.get('Arguments', '')
 
     # Replace argument placeholders for parametric jobs
     # if we have Parameters then we have a parametric job
     if 'Parameters' in parameters:
-      for par, value in parameters.iteritems():
+      for par, value in parameters.items():
```

Comment:
Did not we agree that if we do not want iterator we should use list(xxx.iteritems()) ?

Reply:
https://github.com/DIRACGrid/DIRAC/pull/4243#issuecomment-523920369